pkg/api/utils.go

```diff
@@ -173,6 +172,13 @@ func GetReleases(ctx context.Context, bqc *bqclient.Client) ([]v1.Release, error
 		[]v1.Release{})
 	if len(errs) > 0 {
 		err = errs[0]
+	} else if config != nil {
+		// Add designated releases from config that aren't represented in the DB
+		for name, release := range config.Releases {
+			if release.AddRelease {
```

Comment:
Why does it need to be gated by a boolean? If it's in the config shouldn't we assume it should be shown?